archive/r/r/README.md

```diff
@@ -5,6 +5,7 @@ Welcome to Sample Programs in R!
 ## Sample Programs
 
 - [Capitalize in R][capitalize-article-issue]
+- [Odd-Even in R](https://github.com/TheRenegadeCoder/sample-programs/issues/2297)
```

Comment:
```suggestion
- [Even Odd in R](https://github.com/TheRenegadeCoder/sample-programs/issues/2297)
```

Also, this should be a link to an article issue and follow the linking convention of other programs in the list.